src/main/java/com/google/api/codegen/transformer/java/JavaSchemaTypeNameConverter.java

```diff
@@ -94,9 +91,6 @@ private static String getPrimitiveZeroValue(Schema schema) {
     if (primitiveType.equals("java.lang.String")) {
       return "\"\"";
     }
-    if (primitiveType.equals("com.google.api.gax.httpjson.EmptyMessage")) {
```

Comment:
this was never called (also not called in production, when generating the compute API)